app/views/config_reports/welcome.html.erb

```diff
@@ -8,7 +8,7 @@
   <%= _('If you wish to configure Puppet to forward it reports to Foreman, please follow') %>
   <%= link_to _("setting up reporting"), documentation_url("3.5.4PuppetReports"), :rel => "external" %>
   <%= _("and") %>
-  <%= link_to _("e-mail reporting"), "http://theforeman.org/projects/foreman/wiki/Mail_Notifications", :rel => "external" %>.
+  <%= link_to _("e-mail reporting"), wiki_url("Mail_Notifications"), :rel => "external" %>.
```

Comment:
This does not work, because `wiki_url` is not a helper. It's a controller method. You'd need to extract it to a helper and include that to this controller too. That shouldn't be a hard change, let me know if you need a hand.

Reply:
It should actually be `documentation_url(type: 'wiki', section: 'Mail_Notifications')`

Reply:
I made it `documentation_url('Mail_Notifications', type: 'wiki')` which I think should also work.

Reply:
this renders https://foreman.example.tst/links/manual?section=Mail_Notifications which leads nowhere besides generic manual page. Even when you specify the `section` keyword explicitly, it does not work. The only other link to wiki uses `external_link_path(type: 'wiki')` directly.

Not sure what is broken, my guess is the route definition. Not sure why there's the constraint on section.